README.md

```diff
@@ -77,6 +77,21 @@ passport.use(new wsfedsaml2(
 ));
 ~~~
 
+### Configure strategy for ADFS 
+
+This example utilizes a strategy with SAMLp and RP token encryption.
+
+```javascript
+passport.use('wsfed-saml2', new wsfedsaml2({
+		realm: 'urn:node:samlapp',
+		identityProviderUrl: 'https://my-adfs/adfs/ls',
+        thumbprint: '5D27...D27E',

```

Comment:
If you have access to ADFS server:
1) Open ADFS admin tool
2) AD FS -> Service -> Certificates
3) Select `Token-signing` certificate -> View Certificate...
4) Details -> Copy to File...